### PR TITLE
fixing some inaccuracies in product import documentation

### PR DESCRIPTION
**Pull request by JacobBrownAustin**
## Purpose of this pull request
fixing some inaccuracies in product import documentation
_Describe the goal and the type of changes this pull request covers. Tell us what changes you are making and why._

This pull request (PR) ...

## Magento release version

_Which Magento release(s) are affected by the content changes: 2.4, 2.3? Specify a patch release number, if applicable._
2.4

## Product editions

_Is this update specific to a product edition: Magento Open Source only, Adobe Commerce only?_
no
- ...

_Is this update specific to an installed feature extension: B2B extension, other feature set (please, specify)?_
no
- ...

## Affected documentation pages

_List HTML links for affected pages on <https://docs.magento.com/user-guide/>._
https://experienceleague.adobe.com/docs/commerce-admin/systems/data-transfer/data-csv.html
https://experienceleague.adobe.com/docs/commerce-admin/systems/data-transfer/data-attributes-product.html

- ...
- ...

## Links to Magento source code or PRs

_OPTIONAL. If this pull request references a file in a Magento codebase repository or a code PR, add it here._

- ...
- ...

## Additional information

_OPTIONAL. What other information can you provide?_

### CODE REVIEW

File: src/system/data-attributes-product.md [hunk 1]
```diff
@@ -63,19 +63,19 @@ The installation used to export this data has the sample data installed, and has
...
-|qty_increments|Determines if the product is sold in quantity increments. Options: Yes / No|
-|use_config_enable_qty_inc|Determines if the default configuration setting to enable quantity increments is used, and corresponds to the state of the Use Config Settings checkbox. Values:<br/>**1** — (Yes) The value of the qty attribute is a decimal value.<br/>**0** (or blank) — (No) The value of the qty attribute is a whole number (integer).|
+|use_config_manage_stock|Determines if the default configuration setting for managing stock is used, and corresponds to the state of the Use Config Settings checkbox. Values:<br/>**1** — (Yes)<br/>**0** (or blank) — (No)|
+|use_config_qty_increments|Determines if the default configuration setting for quantity increments is used, and corresponds to the state of the Use Config Settings checkbox. Values:<br/>**1** — (Yes)<br/>**0** (or blank) — (No)|
+|qty_increments|Establishes the number of products that make up a quantity increment.|
```
Comment (JacobBrownAustin) [inline src/system/data-attributes-product.md:77]: For the wording on this,  I just copied it from https://experienceleague.adobe.com/docs/commerce-admin/config/catalog/inventory.html?lang=en .

File: src/system/data-attributes-product.md [hunk 2]
```diff
@@ -63,19 +63,19 @@ The installation used to export this data has the sample data installed, and has
 |use_config_min_qty|Determines if the default value from the configuration is used, and corresponds to the Use Config Settings checkbox. Values:<br/>**1** — (Yes) The default configuration setting is used for the value of this attribute.<br/>**0** (or blank) — (No) The default configuration can be overridden for the value of this attribute.|
 |is_qty_decimal|Determines if the qty attribute has a decimal value. Values:<br/>**1** — (Yes) The value of the qty attribute is a decimal value.<br/>**0** (or blank) — (No) The value of the qty attribute is a whole number (integer).|
 |allow_backorders|Determines if your store allows backorders, and how they are managed.|
-|use_config_backorders|Determines if the default configuration setting for backorders is used, and corresponds to the state of the Use Config Settings checkbox. Values:<br/>**1** — (Yes) The value of the qty attribute is a decimal value.<br/>**0** (or blank) — (No) The value of the qty attribute is a whole number (integer).|
```
Comment (JacobBrownAustin) [inline src/system/data-attributes-product.md:66]: It seems that the documentation for the `is_qty_decimal` field was somehow copied into many of the other fields.

File: src/system/data-csv.md [hunk 3]
```diff
@@ -22,9 +22,9 @@ A full export of the product database contains information about each product in
 
 The first row of the table contains the names of each attribute, which are used as column headers. The remaining rows describe the individual product records. Any row that begins with a value in the SKU column is the beginning of a new product record. A single product might include several rows that contain information about multiple images or product options. The next row that has a value in the SKU column begins a new product.
 
-The category column contains a path for each category to which the product is assigned. The path includes the root category, followed by a forward slash (/) between each level. By default, the pipe “&#124;” character is used to separate different category paths. For example:
```
Comment (JacobBrownAustin) [inline src/system/data-csv.md:25]: I'm not sure why documentation says pipe character.  Did it used to be this way?

Comment (leslietilling) [reply src/system/data-csv.md:25]: I can't really tell when this information was added. Seems like it has been here for a long time, so I am glad you are correcting it.

**Comment by leslietilling** [comment]
@JacobBrownAustin  Sorry for the delay. Looks like this one fell through the cracks as all 2.4 docs are now published on the Experience League and we can no long do updates here. I replicated your changes here: https://git.corp.adobe.com/AdobeDocs/commerce-admin.en/pull/94

Have a look to make sure it is as you intended.

**Comment by leslietilling** [comment]
Replacement PR is merged: https://git.corp.adobe.com/AdobeDocs/commerce-admin.en/pull/94

**Pull request closed without merging**